src/components/LandingPage/AboutEventTile.tsx

```diff
@@ -0,0 +1,87 @@
+import Image from "next/image";
+import type { ReactNode } from "react";
+
+const eventDetailsSectionStyles = "flex flex-col items-center bg-white";
+const eventDetailsContentStyles =
+  "bg-pastel-pink border-[5px] border-dark-pink rounded-[25px] mb-[80px] mt-[50px] md:flex md:justify-center md:max-w-[1100px] md:shadow-[15px_15px_0px_0px_#FF4D6F]";
+const eventImageContainerStyles =
+  "bg-blackish rounded-t-[20px] border-b-[5px] border-dark-pink w-[290px] h-[250px] md:rounded-tr-none md:rounded-l-[20px] md:border-b-0 md:border-r-[5px] md:h-[370px] md:w-[40vw]";
+const eventImageStyles = "w-full h-full object-cover";
+const eventDetailsContainerStyles =
+  "flex items-center w-[290px] h-[370px] rounded-b-[20px] md:w-[50vw] md:rounded-bl-none md:rounded-r-[20px]";
+const eventDetailStyles = "flex items-center ml-[40px]";
+const eventDetailTitleStyles =
+  "text-lg text-blackish font-extrabold leading-[1.2]";
+const iconContainerStyles = "bg-white rounded-[12px] p-[5px] m-[15px]";
```

Comment:
there's a lot of specific numbers here, could we try using the tailwind preset numbers whenever possible?
<img width="796" alt="Screenshot 2024-02-29 at 5 24 50 PM" src="https://github.com/Code-the-Change-YYC/hackathon-management-tool/assets/65151396/b64670c8-dd6a-4797-bd1b-d4b8fe19b09a">

there's a couple numbers that are commonly used but not in the tailwind docs so it might be worth adding to the `tailwind.config.ts` like 5px, 15px and 50px (see [custom width at the bottom](https://tailwindcss.com/docs/width#using-custom-values)